gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/util/DateUtilTest.java

```diff
@@ -0,0 +1,254 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import static uk.nhs.adaptors.pss.translator.utils.IsoDateTimeFormatter.toIsoDateTimeString;
+
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.runner.RunWith;
+
+import junitparams.JUnitParamsRunner;
+import uk.nhs.adaptors.pss.translator.utils.DateFormatUtil;
+
+@RunWith(JUnitParamsRunner.class)
+public class DateUtilTest {
```

Comment:
Would you mind changing these tests to read XML input - just like we did for few other resources in the project?